### PR TITLE
Migration lambda function

### PR DESCRIPTION
**Pull request by william-lau1**
Wrap up all lambda functions for Cognito integration with the current HHAN system. All tests are included in src folders in each lambda function.

*Doubt the Code Inspector's report because it marks the syntax of test codes incorrect and config files in different lambda functions duplicate.

### CODE REVIEW

File: HHAN_Edit_Function/README.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+## Deploy the sample application
+
+The Serverless Application Model Command Line Interface (SAM CLI) is an extension of the AWS CLI that adds functionality for building and testing Lambda applications. It uses Docker to run your functions in an Amazon Linux environment that matches Lambda. It can also emulate your application's build environment and API.
+
+To use the SAM CLI, you need the following tools.
+
+* SAM CLI - [Install the SAM CLI](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html)
+* Node.js - [Install Node.js 10](https://nodejs.org/en/), including the NPM package management tool.
+* Docker - [Install Docker community edition](https://hub.docker.com/search/?type=edition&offering=community)
+
```
Comment (SpencerMarks-IDHA) [inline HHAN_Edit_Function/README.md:29]: clean this up.

File: HHAN_Edit_Function/src/app.js [hunk 2]
```diff
@@ -0,0 +1,95 @@
+const CONFIG = require("./config.json");
+const AWS = require("aws-sdk");
+const HEADER = CONFIG.HEADER;
+const USER_POOL_ID = CONFIG.USER_POOL_TESTID;
+const cognitoidentityserviceprovider = new AWS.CognitoIdentityServiceProvider();
+AWS.config.update({ region: CONFIG.region });
+
+exports.handler = (event, context, callback) => {
+  const body = JSON.parse(event.body);
```
Comment (SpencerMarks-IDHA) [inline HHAN_Edit_Function/src/app.js:9]: document all functions

File: HHAN_Edit_Function/src/app.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+            { Name: "custom:ISADMIN", Value: user.isadmin },
+            { Name: "custom:ORGANIZATION", Value: user.organization },
+            { Name: "custom:TITLE", Value: user.title },
+            { Name: "custom:DEGREE", Value: user.degree },
+            { Name: "custom:ALTEMAIL", Value: user.altemail },
+            { Name: "custom:WORKEMAIL", Value: user.workemail },
+            { Name: "custom:XLS_UPLOAD_PRIV", Value: user.xlsuploadpriv },
+            { Name: "custom:USERNAME", Value: user.username },
+            { Name: "custom:PASSWORD", Value: user.password },
+            // { Name: "email_verified", Value: "true" },
```
Comment (SpencerMarks-IDHA) [inline HHAN_Edit_Function/src/app.js:68]: fix duplicated code
remove commented out code.

File: HHAN_Migration_Function/.idea/vcs.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (SpencerMarks-IDHA) [inline HHAN_Migration_Function/.idea/vcs.xml:1]: do not check in any computer/tool generated files

File: HHAN_Migration_Function/README.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+* **Confirm changes before deploy**: If set to yes, any change sets will be shown to you before execution for manual review. If set to no, the AWS SAM CLI will automatically deploy application changes.
+* **Allow SAM CLI IAM role creation**: Many AWS SAM templates, including this example, create AWS IAM roles required for the AWS Lambda function(s) included to access AWS services. By default, these are scoped down to minimum required permissions. To deploy an AWS CloudFormation stack which creates or modified IAM roles, the `CAPABILITY_IAM` value for `capabilities` must be provided. If permission isn't provided through this prompt, to deploy this example you must explicitly pass `--capabilities CAPABILITY_IAM` to the `sam deploy` command.
+* **Save arguments to samconfig.toml**: If set to yes, your choices will be saved to a configuration file inside the project, so that in the future you can just re-run `sam deploy` without parameters to deploy changes to your application.
+
+You can find your API Gateway Endpoint URL in the output values displayed after deployment.
+
+## Use the SAM CLI to build and test locally
+
```
Comment (SpencerMarks-IDHA) [inline HHAN_Migration_Function/README.md:48]: update

**Review by SpencerMarks-IDHA — CHANGES_REQUESTED** [review]
please address these issue.

**Comment by william-lau1** [comment]
Removed all unnecessary files and documented each lambda function.

*Code inspector review could be ignored safely because it marks config files in each lambda function duplicate, which is not correct.

**Pull request merged**